server/models/db/timeslot.js

```diff
@@ -0,0 +1,20 @@
+const _ = require('lodash');
```

Comment:
Hi Peter,
For greater clarity, I would rename this file to workerTimeslot.js to avoid confusion with the actual Timeslot entity.

Reply:
Done! Thank you for the attention to detail.